src/11_args.py

```diff
@@ -34,14 +34,24 @@ def f2(*args):
 a = [7, 6, 5, 4]
 
 # What thing do you have to add to make this work?
-print(f2(a))    # Should print 22
+# print(f2(a))    # Should print 22
```

Comment:
the solution for this one is a little tricky but its actually staring you in the face. When f2 takes in a variable amount of inputs, you set it up with the *args. All of the examples above line 37 have hard coded multiple inputs, but line 37 is a single input (a list which has multiple items). So because of this, you need to use a * to signal that there are a variable amount of inputs that are going to be received. 

`print(f2(*a))`